### PR TITLE
[WIP] PR: Build macOS app on Big Sur (11.0)

### PR DESCRIPTION
**Pull request by mrclary**
## Description of Changes

Update GitHub action to build on macOS 11.0

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:
@mrclary

### CODE REVIEW

**Comment by mrclary** [comment]
This must wait for resolution of actions/virtual-environments#2486.

File: .github/workflows/installer-macos.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ name: Create macOS App Bundle and DMG
 jobs:
   build:
     name: macOS App Bundle
-    runs-on: macos-latest
+    runs-on: macos-11.0
```
Comment (ccordoba12) [inline .github/workflows/installer-macos.yml:12]: I think we should add a new workflow for Big Sur. Otherwise, we'll affect people still running Catalina.

Comment (mrclary) [reply .github/workflows/installer-macos.yml:12]: Agreed. This is just for testing right now. When this becomes available, and we can build on Apple silicon, then we can consider adding separate workflows or expand the matrix workflow; in any case we should always have a Catalina build

**Comment by mrclary** [comment]
Closing since our application seems to be working on Big Sur

**Pull request closed without merging**